homeassistant/components/prometheus/__init__.py

```diff
@@ -513,6 +513,15 @@ def _handle_automation(self, state):
 
         metric.labels(**self._labels(state)).inc()
 
+    def _handle_counter(self, state):
+        metric = self._metric(
+            "counter",
```

Comment:
Perhaps `counter_value` might be a better name that would allow in the future to add other metrics for the `counter` domain without making this one ambigious?

Reply:
Agreed, this makes sense. Pushed reworked code.